R/hypo.temperature.R

```diff
@@ -0,0 +1,25 @@
+#' @title Get volumetrically averaged hypolimnion temp
+#' @details Calculates volumetrically weighted average hypolimnetic temperature using
+#' the supplied water temperature timeseries. If the lake is not stratified, an NA
+#' value is returned. 
+#'
+#' @param wtr a numeric vector of water temperature in degrees C.
+#' @param depths a numeric vector corresponding to the depths (in m) of the wtr measurements
+#' @param bthA a numeric vector of cross sectional areas (m^2) corresponding to bthD depths
+#' @param bthD a numeric vector of depths (m) which correspond to areal measures in bthA
+#'
+#'
+#' @seealso \code{\link{sim_metrics}}, \code{\link{compare_to_field}}, \code{\link{validate_sim}}
+#'
+#' @export
+hypo.temperature <- function(wtr, depths, bthA, bthD){
+	
+	md = rLakeAnalyzer::meta.depths(wtr, depths)
```

Comment:
Should it be possible to pass (through `...`?) the other params to `meta.depths`?